cluster-operator/src/test/java/io/strimzi/operator/cluster/operator/assembly/KafkaAssemblyOperatorTest.java

```diff
@@ -139,26 +131,75 @@
 import static org.mockito.ArgumentMatchers.anyLong;
 import static org.mockito.ArgumentMatchers.anyString;
 import static org.mockito.ArgumentMatchers.eq;
-import static org.mockito.ArgumentMatchers.isNull;
+import static org.mockito.ArgumentMatchers.startsWith;
 import static org.mockito.Mockito.doAnswer;
 import static org.mockito.Mockito.when;
 
 @ExtendWith(VertxExtension.class)
 @SuppressWarnings({"checkstyle:ClassDataAbstractionCoupling", "checkstyle:ClassFanOutComplexity"})
 public class KafkaAssemblyOperatorTest {
-    public static final Map<String, Object> METRICS_CONFIG = new HashMap<>();
-    public static final InlineLogging LOG_KAFKA_CONFIG = new InlineLogging();
-    public static final InlineLogging LOG_ZOOKEEPER_CONFIG = new InlineLogging();
-    public static final InlineLogging LOG_CONNECT_CONFIG = new InlineLogging();
+    private final static String NAMESPACE = "my-namespace";
+    private final static String CLUSTER_NAME = "my-cluster";
+    private final static MockCertManager CERT_MANAGER = new MockCertManager();
+    private final static PasswordGenerator PASSWORD_GENERATOR = new PasswordGenerator(10, "a", "a");
```

Comment:
This is definitely more a nit, but we should try to be consistent with keyword orderings... I think `private static final` is what is used normally

Reply:
I changed it here. However,  each variant is used 100+ times just here in the assembly operator tests.